build-amp-image.sh

```diff
@@ -78,6 +78,11 @@ if [ $? -ne 0 ]; then
   echo "failed"
   exit 1
 fi
+$DOCKER cp amp-builder:/go/bin/amp-haproxy ./images/haproxy/amp-haproxy.alpine >&2
```

Comment:
ok, I read it too quickly. Anyway you're right, we'll make sure it works as wanted with the new Makefile